estate-services/src/main/java/org/egov/ps/service/PropertyService.java

```diff
@@ -60,7 +60,7 @@
 		propertyValidator.validateUpdateRequest(request);
 		enrichmentService.enrichPropertyRequest(request);
 		String action = request.getProperties().get(0).getAction();
-		if (config.getIsWorkflowEnabled() && !action.contentEquals("") && !action.contentEquals("DRAFT")) {
+		if (config.getIsWorkflowEnabled() && !action.contentEquals("") && !action.contentEquals("DRAFT") && !action.contentEquals("APPROVE")) {
```

Comment:
Can we move the strings `DRAFT` and `APPROVE` to constants?

Reply:
Made the changes.